src/applications/facility-locator/components/FacilityTypeDropdown.jsx

```diff
@@ -80,26 +80,34 @@ class FacilityTypeDropdown extends Component {
           return (
             <div>
               <label
+                aria-live="polite"
+                aria-relevant="additions"
                 htmlFor="facility-dropdown-toggle"
                 id="facility-dropdown-label"
               >
                 Search for
                 {selectedItem === LocationType.CC_PROVIDER && (
                   <span className="cc-info-link-span">
-                    <a
+                    <button
                       onClick={this.toggleCCInfo}
                       title="Click for More Information"
+                      aria-label="Click for more information about the Community Care Program"
                       rel="noopener noreferrer"
                       target="_blank"
                       className="cc-info-link"
                     >
                       <i className="fa fa-info-circle cc-info-link-icon" />
-                    </a>
+                    </button>
                   </span>
                 )}
                 {this.state.showBubble && (
                   <div id="infoBubble">
-                    <span onClick={this.toggleCCInfo}>X</span>
+                    <button
+                      className="ccInfoCloseBtn"
+                      onClick={this.toggleCCInfo}
+                    >
+                      X
```

Comment:
On a screen reader, this will read out the X, which is confusing. This is the markup for a close button we have on our regular modals:

```js
<button
  className="va-modal-close"
  type="button"
  aria-label="Close this modal"
  onClick={this.handleClose}>
  <i className="fa fa-close"></i>
</button>
```
If you don't want to use an icon there, I would wrap the `X` in a span and add `aria-hidden="true"` to it, so screen readers ignore it. And either way, you should add and `aria-label` to the button so that screen readers read out what the button will actually do.

Reply:
Wow, saving this tidbit in my notes. Thanks, @jbalboni